src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -59,57 +63,40 @@ export const SavedSearchAlertListItem: React.FC<SavedSearchAlertListItemProps> =
           flex={1}
           overflow="hidden"
         >
-          <Text
-            variant={["md", "lg"]}
-            color={variant === "active" ? "blue100" : "black100"}
-            style={{ overflowWrap: "break-word" }}
-          >
-            {item.displayName}
-          </Text>
+          <Flex flexDirection="column">
+            <TextWrapper textVariant="lg-display">{item.title}</TextWrapper>
```

Comment:
This could then look like this:

```suggestion
const textColor = ["black100", variant === "active" ? "black100" : "black60"]

.....
.....

            <Text textVariant="lg-display" color={textColor}>{item.title}</TextWrapper>
```